src/app/conf/_components/grid-button.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+type GridButtonProps = {
+  title: string;
+  href: string;
+  disabled?: boolean
+};
+
+const GridButton: React.FC<GridButtonProps> = ({ title, href, disabled }) => {
+  return (
+      <div className="relative w-full h-[500px] flex items-center justify-center overflow-hidden">
+      <div
+        className="grid gap-0 md:scale-100 scale-75"
+        style={{
+          gridTemplateColumns: "repeat(18, 60px)",
+          gridTemplateRows: "repeat(6, 60px)",
+          width: "1080px",
+          height: "360px",
+          boxSizing: "border-box",
+        }}
+      >
+        {Array.from({ length: 18 * 5 + 2 }).map((_, index) => (
+          <div
+            key={index}
+            className="border border-white/10"
+            style={{
+              width: "60px",
+              height: "60px",
+            }}
+          ></div>
+        ))}
+        <a href={disabled ? undefined : href} className="relative z-0 col-span-8 row-span-2 text-white text-3xl font-semibold bg-[#E10098] hover:bg-[#ef00a3] flex items-center justify-center gap-2"
```

Comment:
`<a>` without href is not good for SEO

better to use `const ComponentToUse = disabled ? 'span' : 'a'`